olp-cpp-sdk-core/include/olp/core/client/OlpClientSettings.h

```diff
@@ -151,6 +153,13 @@ struct RetrySettings {
    */
   using BackdownPolicy = std::function<int(int)>;
 
+  /**
+   * @brief A strategy for calculating retry timeouts based on
+   * the initial backdown duration in milliseconds and retry count.
```

Comment:
```suggestion
   * the initial backdown duration and retry count.
```

Reply:
Disagree - I think it is good idea to emphasize that we work with milliseconds.

Reply:
It's becoming bikeshedding :) So I'm okay with this. 
Just add that it's a bit against DRY (repeats what the type already says) and YAGNI (too many details) principles.

Reply:
Ok, I'll remove it. :)